app.yaml

```diff
@@ -13,7 +13,7 @@ handlers:
   upload: build/index.html
   secure: always
   http_headers:
-    X-Frame-Options: "DENY"
+    X-Frame-Options: "SAMEORIGIN"
```

Comment:
@dinvlad I want to get your thoughts on this. The new auth library renders an iframe to do silent token refreshes, which was being blocked on AppEngine. I confirmed this change fixes it. The original PR to add `X-Frame-Options: DENY` is here: https://github.com/DataBiosphere/terra-ui/pull/1935.

Does AppSec have any concerns about relaxing this to `SAMEORIGIN`?

Reply:
@sarahgibs approved this in slack https://broadinstitute.slack.com/archives/CADU7L0SZ/p1652276452198059